nuget/lib/dependabot/nuget/file_fetcher.rb

```diff
@@ -117,7 +117,9 @@ def directory_build_files
             "Directory.Build.props",
             "Directory.build.props",
             "Directory.Build.targets",
-            "Directory.build.targets"
+            "Directory.build.targets",
+            "Packages.props",
```

Comment:
Do we need to look for this file everywhere a project file is present, or just in the root directory? It doesn't look like either case would have found https://github.com/OmniSharp/omnisharp-roslyn/blob/master/build/Packages.props because it's in a `build` folder...

Reply:
Also, do we need to look for the lowercased file?

Reply:
It is just the root next to the `.git` directory. And yes we may need the lowercase.